File: droame booking.js
const addBookingBtn = document.querySelector('#bookings button');

addBookingBtn.addEventListener('click', () => {
  // code to open the booking form
});
const editBtns = document.querySelectorAll('button[type="edit"]');

editBtns.forEach(btn => {
  btn.addEventListener('click', () => {
    // code to open the editing form for this booking or customer
  });
});
const deleteBtns = document.querySelectorAll('button[type="delete"]');

deleteBtns.forEach(btn => {
  btn.addEventListener('click', () => {
    // code to delete this booking or customer from the table
  });
});
const navLinks = document.querySelectorAll('nav a');
const sections = document.querySelectorAll('main section');

navLinks.forEach(link => {
  link.addEventListener('click', (event) => {
    event.preventDefault();
    const sectionId = link.getAttribute('href');
    const sectionToShow = document.querySelector(sectionId);
    sections.forEach(section => section.style.display = 'none');
    sectionToShow.style.display = 'block';
  });
});

File: droame booking.py
from selenium import webdriver

# Open the browser and navigate to the webpage
driver = webdriver.Chrome()
driver.get("C:\Users\yuvak\OneDrive\Desktop\drome bookings.html")

# Find the add booking button and click it
addBookingBtn = driver.find_element_by_css_selector('#bookings button')
addBookingBtn.click()

# Find all the edit buttons and add event listeners to them
editBtns = driver.find_elements_by_css_selector('button[type="edit"]')
for btn in editBtns:
    btn.click()

# Find all the delete buttons and add event listeners to them
deleteBtns = driver.find_elements_by_css_selector('button[type="delete"]')
for btn in deleteBtns:
    btn.click()

# Find all the navigation links and sections and add event listeners to them
navLinks = driver.find_elements_by_css_selector('nav a')
sections = driver.find_elements_by_css_selector('main section')

for link in navLinks:
    link.click()
    sectionId = link.get_attribute('href')
    sectionToShow = driver.find_element_by_css_selector(sectionId)
    sections = driver.find_elements_by_css_selector('main section')
    for section in sections:
        section.style.display = 'none'
    sectionToShow.style.display = 'block'
